.circleci/config.yml

```diff
@@ -938,25 +963,43 @@ workflows:
   launch-all-app-builds-workflow:
     # using do-builds param to select which of the two workflows using the build-on-tag-filters
     # should run; should be this one that just launches the builds or the one actually doing the builds
-    unless: << pipeline.parameters.do-builds >>
+    when:
+      and:
+        - not: << pipeline.parameters.do-builds >>
+        - equal: [ UNKNOWN, << pipeline.parameters.api_call >> ]
     jobs:
-      - conditionally-launch-build-and-store-all-job: &build-and-store-filters
```

Comment:
Note, I choose not to delete unused job `conditionally-launch-build-and-store-all-job` and command `conditionally-launch-build-and-store-all` in this PR because I may need them to investigate any issue in future.